<ui version="4.0" >
 <class>PrefsSQLEditorWidget</class>
 <widget class="QWidget" name="PrefsSQLEditorWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>524</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="titleLabel" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>SQL Editor</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>&amp;Font:</string>
       </property>
       <property name="buddy" >
        <cstring>fontComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="fontComboBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
       <property name="fontFilters" >
        <set>QFontComboBox::MonospacedFonts</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fontSizeSpin" >
       <property name="suffix" >
        <string> pt</string>
       </property>
       <property name="minimum" >
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="useActiveHighlightCheckBox" >
     <property name="text" >
      <string>Use &amp;Active Line Highlighting:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="activeHighlightButton" >
     <property name="text" >
      <string>Color...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="useTextWidthMarkCheckBox" >
     <property name="text" >
      <string>Use &amp;Maximum Text Width Mark:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="textWidthMarkSpinBox" >
     <property name="maximum" >
      <number>139</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>75</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="useCompletionCheck" >
     <property name="text" >
      <string>Use &amp;Code Completion With Length:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QSpinBox" name="completionLengthBox" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QCheckBox" name="useShortcutsBox" >
     <property name="text" >
      <string>Use Editor &amp;Shortcuts:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QPushButton" name="shortcutsButton" >
     <property name="text" >
      <string>&amp;Define...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Syntax Colors</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item rowspan="5" row="0" column="0" >
       <widget class="SqlEditorWidget" name="syntaxPreviewEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="syDefaultButton" >
        <property name="text" >
         <string>Default...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="syKeywordButton" >
        <property name="text" >
         <string>Keywords...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="syNumberButton" >
        <property name="text" >
         <string>Numbers...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="syStringButton" >
        <property name="text" >
         <string>Strings...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="syCommentButton" >
        <property name="text" >
         <string>Comments...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>381</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqlEditorWidget</class>
   <extends>QTextEdit</extends>
   <header>sqleditorwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
